coinprice/src/utils.ts

```diff
@@ -0,0 +1,26 @@
+export function cacheKey() {
+  const date = new Date();
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1;
+  const day = date.getDate();
+
+  return `${year}-${month}-${day}`;
+}
```

Comment:
The one-day caching period is too long for the token price. How about caching it per hour ?

Reply:
1 hour sounds reasonable to me

Reply:
updated to hourly